sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CertificateValidationCheck.vb

```diff
@@ -21,9 +21,9 @@ Namespace Tests.TestCases
 
         Private Sub DirectAddHandlers()
             'Inline version
-            'Secondary@+1
-            CreateRQ().ServerCertificateValidationCallback = Function(sender, certificate, chain, SslPolicyErrors) True         'Noncompliant  {{Enable server certificate validation on this SSL/TLS connection}}
-            '          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                         ^^^^
+            CreateRQ().ServerCertificateValidationCallback = Function(sender, certificate, chain, SslPolicyErrors) True
+            '          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ {{Enable server certificate validation on this SSL/TLS connection}}
+            '                                                                                                      ^^^^ Secondary@-1
```

Comment:
Having all the IDs as [x2] passes fine. Is this desired behavior?
```
        static bool Sec(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)        {
            return true;    //Secondary [x2, x2]
        }
        void Check()        {
            CreateRQ().ServerCertificateValidationCallback += Sec;    //Noncompliant [x2]
            CreateRQ().ServerCertificateValidationCallback += Sec;    //Noncompliant [x2]
        }
```

Reply:
This seems weird. I would say that 2 primary issues should not have the same id, otherwise it looses the purpose of assigning ids to issues